mantidproject/mslice
# Remembering cut axes in non-PSD mode

Addresses some problems with the cut widget in non-PSD mode - mainly the selected axes for a given workspace not being cached correctly.

**To test:**
- Load two **non-PSD** workspaces.
- Select one, switch to cut tab, change the axes and enter some values.
- Select the other workspace and then the original again.
- The cut tab should still be selected and have the axes and values that you entered.

Fixes #297
